nupic/research/frameworks/vernon/mixins/__init__.py

```diff
@@ -55,3 +55,4 @@
 from .update_boost_strength import UpdateBoostStrength
 from .update_dendrite_boost_strength import UpdateDendriteBoostStrength
 from .vary_batch_size import VaryBatchSize
+from .gradient_metrics import GradientMetrics
```

Comment:
(very minor comment) I just sorted this init file alphabetically in my last merge, so I ask that you keep it that way. You can simply run the `isort` command on the file from the command line and it will do it automatically for you.